### PR TITLE
fixed solve attempt command

### PR DESCRIPTION
**Pull request by jboretsky**
trims white space after attempt and allows for a space after !

### CODE REVIEW

File: scripts/trivia.coffee [hunk 1]
```diff
@@ -48,8 +48,8 @@ module.exports = (robot) ->
     
   #solve attempt matches anything after !, case insensitive
   #need to add some way to score points
-  robot.respond /!(.+)$/i, (msg) ->
-    solveAttempt = msg.match[1]
+  robot.respond /!\s?(.+)$/i, (msg) ->
```
Comment (andreirtaylor) [inline scripts/trivia.coffee]: If you make the regex to /! \s+(. +)/i it will match any random number of spaces.

**Comment by andreirtaylor** [comment]
@jboretsky can we close this pull request then?

**Comment by jboretsky** [comment]
Yeah go ahead

**Comment by andreirtaylor** [comment]
Changes are merged into #12

**Pull request closed without merging**